app/src/main/res/layout/fragment_go_live.xml

```diff
@@ -70,14 +72,26 @@
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toBottomOf="@+id/livestream_options_select_channel_text" />
 
+        <TextView
+            android:id="@+id/livestream_options_channel_error_text"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="16dp"
+            android:paddingEnd="16dp"
+            android:visibility="gone"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/livestream_options_select_channel_spinner"
+            tools:visibility="visible" />
```

Comment:
could you add `tools:text=""`with some sample error message? At design time, developers could change the `tools:visibility` to gone, as you have already added it

Reply:
Done in https://github.com/OdyseeTeam/odysee-android/pull/167/commits/7bc99fa92a4340aeccef993c32d0817676efa2c1